x-pack/plugins/apm/public/utils/get_path_for_feedback.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const shortenPath = (path: string, pathStart: string) => {
+  if (path.startsWith(pathStart)) {
+    return path;
+  }
+  const indexOfPathStart = path.indexOf(pathStart);
+  return path.substring(indexOfPathStart);
+};
+
+export const getPathForFeedback = (path: string) => {
+  const pathStartingFromApp = shortenPath(path, '/app');
```

Comment:
Is there a safer way to get this `/app` ? I tried to find some constant or service that would provide this base path, but couldn't find any

Reply:
I couldn't find it as a constant, I see it is used as a string in other places. I defined a constant here so we can reuse it in case we need to have it somewhere else